### PR TITLE
fix: forming Response Files link in the report

### PR DESCRIPTION
**Pull request by DmytroAlipov**
**TL;DR -** This is a fix forming "Response Files" link in the reports (in the context of the ORA functionality).

**What changed?**
The link in the formed report doesn't allow checking the uploaded file in the installation server for Instructor:

image-12

image-9

image-10

Correspond link allows to download file for instructor:

image-11

I rechecked S3 storage forming link too. It works correctly.

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by itsjeyd** [comment]
@DmytroAlipov Thanks for the contribution! Is it ready for review?

**Comment by DmytroAlipov** [comment]
Hi @itsjeyd. Yes of course ;)

**Comment by itsjeyd** [comment]
@DmytroAlipov OK great, we'll get tests enabled for you next.

**Comment by e0d** [comment]
@DmytroAlipov it looks like there's a unit test error that needs attention.

**Comment by DmytroAlipov** [comment]
@e0d Hi! There was a small problem with linting due to an unused variable, but I have fixed it. Could you please run the tests again?

**Comment by e0d** [comment]
@DmytroAlipov We are 🟢

**Comment by itsjeyd** [comment]
Hi @mattcarter, this PR is ready for review by the Aurora team.

**Review by mattcarter — APPROVED** [review]
LGTM 👍🏻
